helpers.ts

```diff
@@ -25,18 +25,18 @@ export type GetParamsOptions = GetQueryOptionsBase | GetQueryOptionsAsMap;
 export function getQuery(
   ctx: Context | RouterContext,
   options: GetQueryOptionsAsMap,
-): Map<string, string>;
+): Map<string, string | undefined>;
```

Comment:
This is unnecessary, as `Map` is possibly undefined in strict mode already.

Reply:
But if we update the type of `result` to `Record<string, string | undefined>`, then the `new Map(Object.entries(result))` will also return a value of `Map<string, string | undefined>`, which is not assignable to `Map<string, string>`.